install_scripts/install.sh

```diff
@@ -10,6 +10,6 @@ mkdir redolog
 mkdir undolog
 
 chown -R mysql:mysql *
-sh ./bin/mysqld --defaults-file=./stonedb.cnf --initialize
+./bin/mysqld --defaults-file=./stonedb.cnf --initialize
```

Comment:
Good job!
 One step further, change `stonedb.cnf` to `my.cnf`, you can backport code from pr:https://github.com/stoneatom/stonedb/issues/117